src/csharp/Microsoft.Spark.UnitTest/Sql/ColumnTests.cs

```diff
@@ -363,6 +363,68 @@ public void TestOver()
             }
         }
 
+        [Fact]
+        public void TestIsIn()
+        {   
+            {
+                var expected = new System.Collections.Generic.List<string> {"vararg_1", "vararg_2"};
+                Column column1 = CreateColumn("col1");                
```

Comment:
You can rename all `column1` to `column`.

Reply:
I just wanted to check you did want this as the existing tests used the same Column column1 = CreateColumn("col1") - even when the tests just create one column.

Let me know if you do want me to rename it in the IsIn tests as well :)

Reply:
Ok it is already existing tests, then it's fine. I will clean up later. Thanks!

Reply:
great - if you raise an issue and say what you want (if one column no number, what if two columns or more, start at 1 then) am happy to do it as a separate pr